### DISCUSSION TITLE
Support for Bubble.io single-page application?

### DISCUSSION THREAD

**Discussion by ricotrevisan** [plausible/analytics#1196] [Anything Else]
⚠️ Rookie / non-coder warning

I built my application using [Bubble.io](https://bubble.io/?ref=gkkmjiny) using their single-page functionality. When I look at my dashboard I see only the root page and not the actual item. I see:

`https://refactoringnocode.com/dashboard`

Instead of seeing:

`https://refactoringnocode.com/dashboard?mode=how-tos&item=designing-in-bubble`

I've tried both the normal tracker and the hash-based one (following Carrd's instructions). Any clues on how to make it work on Bubble.io?

**Comment by ukutaht** [comment]
Hey @RicoTrevisan. Looks like Bubble uses [query parameters](https://en.wikipedia.org/wiki/Query_string) for navigation. Plausible currently does not record query parameters at all so it's not possible to get the full URL.

It's very unusual to implement navigation using query parameters which is why we don't record them by default. For most people it would create more noise than be helpful. Maybe Bubble has a good reason for using query parameters. Anyways, this is a [highly requested feature](https://github.com/plausible/analytics/discussions/177) but unfortunately I cannot give you a timeline for when it will be ready.

**Reply by ricotrevisan** [reply]
Thanks! In the meantime, I've been discussing with other Bubbler's are there are ways of moving away from parameters-based navigation. It's not native, but there are ways around it.
